### PR TITLE
Version next controls

### PR DESCRIPTION
**Pull request by Wagnerp**

### CODE REVIEW

**Comment by Wagnerp** [comment]
I have no idea what's causing the conflicts.

File: Source/Krypton Toolkit/Examples/Controls/Form1.cs [hunk 1]
```diff
@@ -11,7 +10,7 @@
 
 namespace Controls
 {
-    public partial class Form1 : KryptonForm
+    public partial class Form1 : Form
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Examples/Controls/Form1.cs]: Why not `Krypton Form`?

File: Source/Krypton Toolkit/Examples/Controls/Controls.csproj [hunk 2]
```diff
@@ -1,22 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0-windows</TargetFramework>
-    <LangVersion>latest</LangVersion>
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Examples/Controls/Controls.csproj:5]: Keep it set to the `Latest`

File: Source/Krypton Toolkit/Examples/Controls/Controls.csproj [hunk 3]
```diff
@@ -1,9 +1,26 @@
...
     <OutputType>WinExe</OutputType>
-    <TargetFramework>net6.0-windows</TargetFramework>
     <UseWindowsForms>true</UseWindowsForms>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <Prefer32Bit>true</Prefer32Bit>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Examples/Controls/Controls.csproj]: Remove the bitness of the build!

File: Source/Krypton Toolkit/Examples/Controls/Controls.csproj [hunk 4]
```diff
@@ -1,9 +1,26 @@
...
+    <Prefer32Bit>true</Prefer32Bit>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Krypton.Toolkit.Canary" Version="6.0.2103.6" />
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Examples/Controls/Controls.csproj]: Remove this, as it should be controlled by the inclusion from the extended project (i.e. remove conflicts)

File: Source/Krypton Toolkit/Examples/Controls/Form1.cs [hunk 5]
```diff
@@ -1,4 +1,5 @@
-﻿using System;
+﻿using Krypton.Toolkit;
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Examples/Controls/Form1.cs]: Sorting.

**Review by Smurf-IV — COMMENTED** [review]
> 
> 
> I have no idea what's causing the conflicts.

_Might_ be down to the other "Unresolved" merges, and you are "Moving on" without taking a new branch from version-next

File: Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Controls/Readme.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+﻿
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Controls/Readme.md:1]: This still needs to be refereced from the Documents file

**Comment by Smurf-IV** [comment]
@Wagnerp Still has conflicts and requires doc updates

**Comment by Wagnerp** [comment]
> @Wagnerp Still has conflicts and requires doc updates

@Smurf-IV That'll be Thursdays/Fridays task.

**Comment by Wagnerp** [comment]
Should be ready to go!

**Pull request merged**